perfkitbenchmarker/static_virtual_machine.py

```diff
@@ -29,8 +29,13 @@
 import logging
 import threading
 
+from perfkitbenchmarker import flags
 from perfkitbenchmarker import linux_virtual_machine
 from perfkitbenchmarker import virtual_machine
+from perfkitbenchmarker import windows_virtual_machine
+
+WINDOWS = 'windows'
```

Comment:
Doesn't need to be during this PR, but this could be a bit more centralized. DEBIAN and RHEL are now defined in benchmark_spec.py, package_managers.py, and static_virtual_machine.py. I suspect WINDOWS will be needed elsewhere too.